src/test/java/FunctionalTest/LoginPageTest.java

```diff
@@ -0,0 +1,29 @@
+package FunctionalTest;
+
+import Base.BaseTest;
+import PageObjects.LoginPage;
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+public class LoginPageTest extends BaseTest {
+
+@Test
+   private void shouldBeAbleToLoginWithValidUserNameAndPassword()  {
+
+
+    driver.get("https://www.docusign.com/");
+    driver.manage().window().maximize();
+    LoginPage login=new LoginPage(driver);
+    login.clickLoginLink();
+    login.enterMailId("taruna.cotti@gmail.com");
+    login.clickOnNext();
+    login.enterPassword("TestPractice@2");
+    login.clickOnNext();
+   // String loginVerification=driver.findElement(By.xpath("//button[@data-qa='header-HOME-tab-button']")).getText();
```

Comment:
never leave unused code, if you have unfinished code below approach is advised
//TODO: Some work is pending here, so you can come back and finish